lib/Core/ExecutorUtil.cpp

```diff
@@ -59,6 +59,14 @@ namespace klee {
       } else if (isa<ConstantPointerNull>(c)) {
         return Expr::createPointer(0);
       } else if (isa<UndefValue>(c) || isa<ConstantAggregateZero>(c)) {
+        if (getWidthForLLVMType(c->getType()) == 0) {
+          if (isa<llvm::LandingPadInst>(ki->inst)) {
+            klee_warning_once(
+                0,
+                "Using zero size array fix for landingpad instruction filter");
```

Comment:
@futile sorry to be pedantic here, and I know that I asked you to run `clang-format`, but I typically reverse these kind of breaks that `clang-format` inserts, as I find them ugly and inconsistent with the rest of the code.  Can you have `klee_warning_once` on a single line?  I'll then merge it.

Reply:
Okay, worked on it!